docs/src/components/landing/Banner.js

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import Typography from '@mui/material/Typography';
+import Stack from '@mui/material/Stack';
+import Button from '@mui/material/Button';
+import Link from 'docs/src/modules/components/Link';
+import KeyboardArrowRightRounded from '@mui/icons-material/KeyboardArrowRightRounded';
+
+export default function Banner({ content }) {
+  return (
+    <Container
+      sx={{
+        pt: 2,
+        pb: { xs: 2, sm: 4, md: 8 },
+        scrollMarginTop: 'calc(var(--MuiDocs-header-height) + 32px)',
+      }}
+      id="banner-full-width"
+    >
+      <Stack
+        sx={{
```

Comment:
not sure what our practices usually are about this, but maybe it would be nice to have the styled object outside of the component somehow, or use styled components, so that we don't mix lots of styles and JSX too much?